src/test/kotlin/com/linchpino/core/controller/AccountControllerTestIT.kt

```diff
@@ -78,7 +78,7 @@ class AccountControllerTestIT {
 
     @Test
     fun `test creating account with blank lastName results in bad request`() {
-        val invalidRequest = CreateAccountRequest("John", "", "john.doe@example.com", "secret", 1)
+        val invalidRequest = CreateAccountRequest("John", "", "john.doe@example.com", "secret", 2)
```

Comment:
Maybe it would be better to use "AccountTypeEnum.MENTOR.value" instead of number 2